src/app/components/url-box/url-box.component.html

```diff
@@ -107,6 +107,8 @@
   >
     <span *ngIf="isLoading">{{ 'LOADING_INDICATOR_TEXT' | translate }}</span>
     <span *ngIf="!isLoading">{{ 'SEND_REQUEST_BUTTON' | translate }}</span>
+    <span class="button-tip-label" *ngIf="!isLoading && isMac">[{{sendRequestKeybinds[0]}}]</span>
+    <span class="button-tip-label" *ngIf="!isLoading && !isMac">[{{sendRequestKeybinds[1]}}]</span>
```

Comment:
Tbh I'm still not comfortable with this, but I can't come up with a better approach for now.

Reply:
Also for this, I think it would be better if the tip label is hidden by default and only shows (ease in) when the user hovers over the send button.